discovery_windows.go

```diff
@@ -61,6 +61,10 @@ func systemDir() string {
 	return filepath.VolumeName(cwd) + "\\"
 }
 
-func userProfileDir() string {
-	return os.Getenv("USERPROFILE")
+func userHomeDir() string {
```

Comment:
I don't think having this function is useful but it's not a big deal either

Reply:
I don't mind either way either I was just following the convention of the way the system dir is fetched.